source/technical/swift.rst

```diff
@@ -203,36 +203,76 @@ ________________________________________
 Mounting Object Store as a File System
 ________________________________________
 
-When logged into an instance using Chameleon supported images, such as ``CC-CentOS7`` and ``CC-Ubuntu16.04``, you may use the pre-installed ``cloudfuse`` (Source: `Github <https://github.com/redbo/cloudfuse>`_) to mount your Chameleon Object Store as a directory on your Linux environment.
-
-Before mount, create a ``~/.cloudfuse`` file with the following content:
-
-.. code-block:: bash
-
-   username=<username>
-   password=<password>
-   tenant=<projectname>
-   region=<regionname> # CHI@TACC or CHI@UC
-   authurl=https://chi.tacc.chameleoncloud.org:5000/v2.0
-
-Replace ``username`` and ``password`` with your Chameleon username and password; replace ``projectname`` with your Chameleon project name; and replace ``regionname`` with the regional endpoint to use.
-
-Then mount with the following command:
+When logged into an instance using Chameleon supported images, such as `CC-CentOS7 <https://www.chameleoncloud.org/appliances/1/>`_ and `CC-Ubuntu16.04 <https://www.chameleoncloud.org/appliances/19/>`_, 
+you will see a directory called ``my_mounting_point`` which is a pre-mounted directory to your Chameleon Object Store at the same site of your instance. 
```

Comment:
```suggestion
you will see a directory called ``my_mounting_point`` which is a pre-mounted directory to your Chameleon Object Store at the same site of your instance. Each Object Store container that you have access to will appear as a subdirectory inside this mount.
```

^ Is that right? I forget if there are subdirectories for projects (and then container folders within those) or if the containers themselves are the top-level directory.

Reply:
What you added is correct. The containers are the top-level.